server/helpers/utilities.js

```diff
@@ -98,6 +99,8 @@ const getSnoozeUntil = (rawSnoozeUntil) => {
   return snoozeUntil.isAfter(moment()) ? snoozeUntil.add(1, 'day').format('dddd, Do MMMM YYYY') : ''
 }
 
+const appendUserErrorMessage = (error, userMessage = GENERAL_ERROR) => Object.assign(error, { userMessage })
```

Comment:
Can spread be used here instead of Object.assign?

Not sure if it's just preference thing but I find it more readable with:

`{ ...error,  userMessage }`

Reply:
Controversially, I actively selected against immutability. Assign preserves the existing object. I'm just adding a param via a function for consistency.